### DISCUSSION TITLE
Possibility to cache transformed images to disk

### DISCUSSION THREAD

**Discussion by jobernolte** [coil-kt/coil#1733] [Q&A]
Hi there,

Is there any possibility to cache a transformed image to disk and use this the next time the same request is made? For example:

`
val request = ImageRequest.Builder(context)
  .data("some url")
  .transformations(SomeExpensiveTransformation)
  .build()
`

What I want to do is to save the resulting image after the transformation to disk and when this particularly request is made a second time, the already transformed image is read from the disk cache?

**Comment by colinrtwhite** [comment]
Answered here: https://github.com/coil-kt/coil/issues/1959#issuecomment-1859285041
